package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testengine-cli",
-  "version": "1.5.0",
+  "version": "1.8.0",
```

Comment:
imho, I think this should be 1.6.0 instead
Because the versioning of TestEngine API and TestEngine CLI are not in sync.

Reply:
Was wondering the same as well...  happy to make the change.